config/7.11.0/common.yaml

```diff
@@ -1009,6 +1025,39 @@ servers:
                 #[[end]]
                 #[[end]]
       ## KIE server deployment config END
+    ## KIE server pvc BEGIN
+    #[[if .PersistRepos]]
+    persistentVolumeClaims:
+      - metadata:
+          name: "[[$.ApplicationName]]-m2-repository-claim"
```

Comment:
Persistent volume to store maven artifact is maintained for every `ApplicationName`. Does it possible to have multiple `ApplicationName`? If so does both applications maintain their own maven repo?

Reply:
each deployment config should point to its own PV.